sos/plugins/openstack_glance.py

```diff
@@ -38,15 +39,22 @@ def setup(self):
 
         self.limit = self.get_option("log_size")
         if self.get_option("all_logs"):
-            self.add_copy_spec(["/var/log/glance/",
-                                "/var/log/containers/glance/"],
-                               sizelimit=self.limit)
+            self.add_copy_spec([
+                "/var/log/glance/",
+                "/var/log/containers/glance/"
+            ], sizelimit=self.limit)
         else:
-            self.add_copy_spec(["/var/log/glance/*.log",
-                                "/var/log/containers/glance/*.log"],
-                               sizelimit=self.limit)
-
-        self.add_copy_spec("/etc/glance/")
+            self.add_copy_spec([
+                "/var/log/glance/*.log",
+                "/var/log/containers/glance/*.log"
+            ], sizelimit=self.limit)
+
+        self.add_copy_spec([
+            "/etc/glance/",
+            self.var_puppet_gen + "/etc/glance/",
+            self.var_puppet_gen + "/etc/my.cnf.d/"
```

Comment:
Missing comma or '+' at the end of line? (or missing path for tripleo.cnf?)

Reply:
thx fixed